app/scenes/Document/components/Header.js

```diff
@@ -118,8 +105,6 @@ class Header extends React.Component<Props> {
   };
 
   render() {
-    if (this.redirectTo) return <Redirect to={this.redirectTo} push />;
```

Comment:
Without the remount this state caused problems – why was it built like this instead of a regular link? Must have been having a weird day.